code/style.css

```diff
@@ -2,6 +2,353 @@
+  font-family: 'Alegreya Sans SC', sans-serif;
+  position: absolute;
+  top: 10%;
+  left: 12%;
+  z-index: 1;
+  
+}
+
+.header-text h1 {
+  font-size: 45px;
+  margin: 0;
+}
+
+.nav-bar {
+  display: grid;
+  grid-template-columns: 1fr;
+  border-bottom: black solid 1px;
+  border-top: black solid 1px;
+}
+
+.nav-bar a {
+  color: black;
+  text-decoration: none;
+  align-self: center;
+  padding: 30px;
+}
+
+.nav-bar a:hover {
+  color: white;
+  background-color: slategray;
+}
+
+#menu-button ~ a {
+  display: none;
+}
+
+#menu-button.open ~ a {
+  display: block;
+}
+
+.b-container {
+  width: 100%;
+  margin: 0;
+}
+
+.business-text {
+
+  margin: 0 auto 0 auto;
+  padding-top: 50px;
+  width: 80%;
+  font-style: italic;
+  text-align: center;
+  font-size: 20px;
+  line-height: 1.8;
+}
+
+.main-content {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: minmax(200px, auto);
+  align-items: stretch;
+  justify-items: stretch;
+  gap: 20px;
+  padding: 5%;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  font-size: 20px;
+}
+
+.form, .accordion {
+  background-color:rgba(222, 223, 215, 0.924);
+  margin: 10px;
+  padding: 20px;
+  border: solid black 1px;
+  border-radius: 5px;
+  box-shadow: 3px 6px 5px 1px rgba(8, 8, 8, 0.2);
+}
+
+.form h1 {
+  text-align: center;
+  font-size: 26px;
+  padding: 20px;
+}
+
+.form-labels {
+  margin: 0 20px 20px 20px;
+}
+
+.fieldset {
+  border-radius: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 10px;
+  
+}
+
+.input, .select-input {
+  display: block;
+  line-height: 1.7;
+  border-radius: 5px;
+  padding-left: 5px;
+  width: 200px;
+  font-family: inherit;
+}
+
+input:valid {
+  background-color: rgb(142, 192, 142);
+}
+
+input:invalid {
+  background-color: rgb(243, 187, 155);
+}
+
+.select-input {
+  padding: 2px;
+  margin-top: 1px;
+  width: 210px;
+}
+
+.radio-buttons {
+  padding: 10px;
+}
+
+.radio-buttons > input {
+  display: inline;
+  width: auto;
+}
+
+.radio-buttons > label {
+  margin-right: 30px;
+}
+
+.radio-buttons > label:hover {
+  color: white;
+  background-color: slategray;
+  border-radius: 5px;
+}
+
+.form-labels.check {
+  width: 80%;
+}
+
+.input.checkbox {
+  display: inline;
+  width: auto
+}
+
+#submit-button {
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  font-weight: bolder;
+  width: 50%;
+  margin: 20px auto;
+  border-radius: 5px;
+  box-shadow: 1px 1px 3px slategray;
+  height: 50px;
+}
+
+#submit-confirmation {
+  display: none;
+}
+
+.accordion h1 {
+  text-align: center;
+  font-size: 26px;
+}
+
+.question {
+  display: flex;
+  justify-content: space-between;
+  font-weight: bold;
+  margin-top: 30px;
+  padding: 0 30px;
+  background-color: rgb(244, 245, 238);
+  border: black solid 1px;
+}
+
+.question:hover, #submit-button:hover {
+  color: white;
+  background-color: rgb(66, 39, 7);
+  cursor: url(./pictures/icons8-french-press-48.png), auto;
+}
+
+.question:nth-child(4n+4) {
```

Comment:
:nth-child (4n+4)??? what does that mean? could maybe use a comment just to explain what this does